olive/systems/utils/__init__.py

```diff
@@ -3,11 +3,19 @@
 # Licensed under the MIT License.
 # --------------------------------------------------------------------------
 from olive.systems.utils.arg_parser import get_common_args
-from olive.systems.utils.misc import create_new_system, create_new_system_with_cache, get_package_name_from_ep
+from olive.systems.utils.misc import (
+    create_new_environ,
+    create_new_system,
```

Comment:
the create_new_system doesn't need to be exposed except the testing

Reply:
I didn't add the 'create_new_system'. It was already present.

Reply:
Do you mean 'create_new_environ' that I added?

Reply:
my bad